src/vtvalidate.cpp

```diff
@@ -5,55 +5,71 @@
 #include <iostream>
 #include <map>
 #include <stdexcept>
-
-/**
- * This is an asynchronous standalone function that logs a string.
- * @name helloAsync
- * @param {Object} args - different ways to alter the string
- * @param {boolean} args.louder - adds exclamation points to the string
- * @param {Function} callback - from whence the hello comes, returns a string
- * @returns {string}
- * @example
- * var module = require('./path/to/lib/index.js');
- * module.helloAsync({ louder: true }, function(err, result) {
- *   if (err) throw err;
- *   console.log(result); // => "...threads are busy async bees...hello
- * world!!!!"
- * });
- */
+#include <vtzero/vector_tile.hpp>
 
 namespace VectorTileValidate {
 
-// Expensive allocation of std::map, querying, and string comparison,
-// therefore threads are busy
-std::string do_expensive_work(bool louder) {
+struct geom_handler {
```

Comment:
Our geom handler should not allocate a single thing and just provide methods that do nothing. We do not need to store the output geometry as we are not doing any validation on the geometry itself.

Reply:
> as we are not doing any validation on the geometry itself.

We're not?

Reply:
We are validating that the geometry encoding is not malformed, but the geometry itself we do not analysis on in this process currently. For example we are not running `boost::is_valid` or anything. So we are validating that it is not an improperly encoded geometry but not using the geometry. Therefore, we really do not need to save the points.

Reply:
@flippmoke Also, I honestly dont really understand the need for geom handlers, so I welcome any input on what their purpose is and what youd expect a handler to return/do